crawl-ref/source/spl-miscast.h

```diff
@@ -8,10 +8,10 @@
 #include "spl-util.h"
 
 // scale miscast severity
-const int MISCAST_DIVISOR = 9;
+static const int MISCAST_DIVISOR = 9;
 
 // cutoff for getting a miscast
-const int MISCAST_THRESHOLD = 150;
+static const int MISCAST_THRESHOLD = 150;
```

Comment:
For your future reference `static const` and `const` mean the same thing in C++; crawl code is inconsistent about which one we use (of course).

If you could rebase this dropping the staticify commit (so `git blame` is a bit easier for these lines) this looks good to go.

Reply:
Thanks ebering.

Unlike many of the folks in -dev I don't have formal CS experience, and I sometimes realize I've been cargo culting something or other, so I appreciate your taking the time.

Reply:
Okay, I've rebased to master and dropped the staticify commit.

Reply:
> Unlike many of the folks in -dev I don't have formal CS experience

Even with formal experience C++ is widely known to be a monster.